install-conda-dependencies.sh

```diff
@@ -6,4 +6,4 @@ echo "==== Installing Conda packages for $(which conda)"
 
 conda info --envs
 
-conda install --yes pocl numpy pyvisfile pyopencl islpy
```

Comment:
PyOpenCL and islpy take a bit of time to build, and I expect them to change infrequently. So there's a trade-off of install speed vs. hacking convenience here. I'm OK with leaving them source-built as you have here, since it's just a few minutes of build time, but I figured I would bring up that the option exists.